src/targets/gpu/kernels/include/migraphx/kernels/roialign.hpp

```diff
@@ -199,25 +206,25 @@ __device__ void roialign(const T& x_t, const U& rois_t, const V& ind_t, W& y_t,
         const auto offset_x = x + ((batch_ind * channel_num + c) * in_dims[0] * in_dims[1]);
         if constexpr(s.is_avg_pooling)
         {
-            y_t[i] = calc_pooling(offset_x,
-                                  roi_starts,
-                                  bin_size,
-                                  {ph, pw},
-                                  bin_grid_size,
-                                  in_dims,
-                                  s.roi_offset,
-                                  avg_pool{});
+            y_t[i] = static_cast<ytype>(calc_pooling(offset_x,
```

Comment:
Sanity check question: we're converting from potentially fp8 -> float  -> do the mul for roi_start/end and then convert back to fp8?

Reply:
I guess maybe rebase /retarget to #2455  so one that's done we aren't rereviewing the same thing or stale code.